modules/spring-data-2.0/src/test/java/org/apache/ignite/springdata/misc/PersonRepository.java

```diff
@@ -104,6 +104,10 @@
     @Query("DELETE FROM Person WHERE secondName = ?")
     public void deleteBySecondNameQuery(String secondName);
 
+    /** Delete using @Query */
+    @Query("delete FROM Person WHERE secondName = ?")
+    public void deleteBySecondNameLowerCase(String secondName);
+
     /** Delete using @Query but with errors on the query */
     @Query("DELETE FROM Person WHERE firstName = ? AND ERRORS = 'ERRORS'")
```

Comment:
Let's just convert this line to lower case instead of adding new test and new statement